### ISSUE TITLE
need to escape colons in text

### ISSUE THREAD

**Issue by MichaelTiemannOSC** [os-climate/osc-ingest-tools#26]
The SEC DERA tag table has a row that evalutes to the following VALUES:

('ProvisionForIncomeTaxes', '0001552781-17-000597', True, True, 'monetary', 'D', 'D', 'Provision for income taxes', ':Provision for income taxes.', '2017q4')

When we try to insert those values, the string `:Provision` is treated as a bind parameter.  Is the right place to escape the colon in _sqlform (where we do other string escapes)?  Are there other special characters for SQLAlchemy we have to watch out for?

**Comment by erikerlandson** [comment]
Not sure colons, per se, are the problem, for example I added some colons here and it seemed to work fine:
```
inserting 1 records
  ('2021Q4', 0.6)
constructed fully qualified table name as: "osc_datacommons_dev.sandbox.trino_batch_insert_test"
batch insert result: [(1,)]
inserting 1 records
  ('2021Q4', 0.7)
constructed fully qualified table name as: "osc_datacommons_dev.sandbox.trino_batch_insert_test"
batch insert result: [(1,)]
inserting 1 records
  ('2021:Q4', 0.8)
constructed fully qualified table name as: "osc_datacommons_dev.sandbox.trino_batch_insert_test"
batch insert result: [(1,)]
inserting 1 records
  (':2021:Q4', 0.8)
constructed fully qualified table name as: "osc_datacommons_dev.sandbox.trino_batch_insert_test"
batch insert result: [(1,)]
```

**Comment by erikerlandson** [comment]
xref https://github.com/os-climate/os_c_data_commons/issues/165

**Comment by erikerlandson** [comment]
closed by #27
